packages/math-input/src/components/keypad.tsx

```diff
@@ -5,29 +5,28 @@
 
 import * as React from "react";
 import ReactDOM from "react-dom";
-import {connect} from "react-redux";
 
-import {removeEcho} from "../actions/index";
 import {View} from "../fake-react-native-web/index";
 
 import EchoManager from "./echo-manager";
+import GestureManager from "./gesture-manager";
 import PopoverManager from "./popover-manager";
 
-import type {State} from "../store/types";
 import type {Popover, Echo} from "../types";
 import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
-interface ReduxProps {
-    active: boolean;
-    echoes: ReadonlyArray<Echo>;
-    popover: Popover | null;
-}
-
-interface Props extends ReduxProps {
+type Props = {
     children: React.ReactNode;
     style?: StyleType;
     removeEcho?: (animationId: string) => void;
-}
+    active: boolean;
+    echoes: ReadonlyArray<Echo>;
+    gestureManager: GestureManager;
+    gestureFocus: any;
+    popover: Popover | null;
+    heightPx: number;
+    widthPx: number;
```

Comment:
**suggestion:** people might think that this specifies the width/height of the keypad when in reality it's specifying the dimensions of the buttons.  Maybe these could be group together and the prop renamed, e.g.
```
buttonDimensions: Dimensions,
```
or
```
buttonSize: Size,
```